test-dev/Makefile.vc.in

```diff
@@ -1,7 +1,7 @@
 CC	= cl
 CFLAGS	= /O2 /W3 /MD /I..\include /I..\src /DWIN32 \
 	  /Dinline=__inline /DPATH_MAX=1024 /D_USE_MATH_DEFINES /D_CRT_SECURE_NO_WARNINGS
-LDFLAGS = /RELEASE /OUT:$(EXE)
+LDFLAGS = /Lkernel32.lib /RELEASE /OUT:$(EXE)
```

Comment:
Why is this needed? We are not linking with /nodefaultlib or something, don't we?

Reply:
It fails with undefined reference to `IsDebuggerPresent` if not specify this

Reply:
I added some code to avoid message boxes pop ups when something got failed during the unit test on CI:
https://stackoverflow.com/questions/13943665/how-can-i-disable-the-debug-assertion-dialog-on-windows

Reply:
And, that fails at Autotools bulid, not at CMake build

Reply:
P.S. Exact failure moment I trying to explain:
https://github.com/libxmp/libxmp/runs/3096980093?check_suite_focus=true#step:5:891

Reply:
Ah, I guess, I did another fail: I need to include the <debugapi.h>, otherwise, it makes the wrong symbol that gets undefined, really...

Reply:
Maybe you should include windows.h or something: winbase.h says that IsDebuggerPresent is a stdcall function, so we need is declaration, no?